vm/simple_server.go

```diff
@@ -10,10 +10,10 @@ import (
 	"path/filepath"
 	"unicode"
 
+	"fmt"
 	"github.com/fatih/structs"
 	"github.com/goby-lang/goby/vm/classes"
 	"github.com/gorilla/mux"
-	"fmt"
 	"strconv"
```

Comment:
also grouped with the std lib

Reply:
@ear7h I already ran `gofmt` and I want to follow what it changes/allows

Reply:
Yes, but I believe gofmt preserves newlines so the two groups of imports are formatted independently of each other.

Reply:
If you insist I can change this time, but I don't see the point to group them all manually in every future PRs

Reply:
Alright, makes sense